tensorflow_quantum/core/ops/tfq_unitary_op.py

```diff
@@ -16,42 +16,51 @@
 import tensorflow as tf
 from tensorflow_quantum.core.ops import tfq_utility_ops
 from tensorflow_quantum.core.ops.load_module import load_module
+from tensorflow_quantum.python import quantum_context
 
 OP_MODULE = load_module("_tfq_calculate_unitary_op.so")
 
 
-def calculate_unitary(programs, symbol_names, symbol_values):
-    """Calculate the unitary matrix for the given circuits.
-
+def get_unitary_op(
+        quantum_concurrent=quantum_context.get_quantum_concurrent_op_mode()):
+    """Get an op that calculates the unitary matrix for the given circuits.
 
+    >>> unitary_op = tfq.get_unitary_op()
     >>> qubit = cirq.GridQubit(0, 0)
     >>> symbol = sympy.Symbol('alpha')
     >>> my_circuit = cirq.Circuit(cirq.H(qubit) ** symbol)
     >>> tensor_circuit = tfq.convert_to_tensor([my_circuit])
-    >>> tfq.calculate_unitary(tensor_circuit, ['alpha'], [[0.2]])
+    >>> unitary_op(tensor_circuit, ['alpha'], [[0.2]])
     <tf.RaggedTensor [
         [[(0.9720+0.0860j), (0.0675-0.2078j)],
          [(0.0675-0.2078j), (0.8369+0.5017j)]]]>
 
 
     Args:
-        programs: `tf.Tensor` of strings with shape [batch_size] containing
-            the string representations of the circuits to be executed.
-        symbol_names: `tf.Tensor` of strings with shape [n_params], which
-            is used to specify the order in which the values in
-            `symbol_values` should be placed inside of the circuits in
-            `programs`.
-        symbol_values: `tf.Tensor` of real numbers with shape
-            [batch_size, n_params] specifying parameter values to resolve
-            into the circuits specified by programs, following the ordering
-            dictated by `symbol_names`.
+        quantum_concurrent: Optional Python `bool`. True indicates that the
+            returned op should not block graph level parallelism on itself when
+            executing. False indicates that graph level parallelism on itself
+            should be blocked. Defaults to value specified in
+            `tfq.get_quantum_concurrent_op_mode` which defaults to True
+            (no blocking). This flag is only needed for advanced users when
+            using TFQ for very large simulations, or when running on a real
+            chip.
 
     Returns:
```

Comment:
I think it would be good to expand the docstring to where it is like that for the [other ops](https://github.com/tensorflow/quantum/blob/67faf64d9e984acb29330ccd6bebe0acb463ae47/tensorflow_quantum/core/ops/circuit_execution_ops.py#L93), I find those explicit arg and shape descriptions helpful.

Reply:
Oops, good call. Added.